dapr/proto/common/v1/common_pb2.py

```diff
@@ -26,30 +27,30 @@
   _STATEITEM_METADATAENTRY._serialized_options = b'8\001'
   _CONFIGURATIONITEM_METADATAENTRY._options = None
   _CONFIGURATIONITEM_METADATAENTRY._serialized_options = b'8\001'
-  _HTTPEXTENSION._serialized_start=87
-  _HTTPEXTENSION._serialized_end=295
-  _HTTPEXTENSION_VERB._serialized_start=181
-  _HTTPEXTENSION_VERB._serialized_end=295
-  _INVOKEREQUEST._serialized_start=298
-  _INVOKEREQUEST._serialized_end=448
-  _INVOKERESPONSE._serialized_start=450
-  _INVOKERESPONSE._serialized_end=524
-  _STREAMPAYLOAD._serialized_start=526
-  _STREAMPAYLOAD._serialized_end=568
-  _STATEITEM._serialized_start=571
-  _STATEITEM._serialized_end=819
-  _STATEITEM_METADATAENTRY._serialized_start=772
-  _STATEITEM_METADATAENTRY._serialized_end=819
-  _ETAG._serialized_start=821
-  _ETAG._serialized_end=842
-  _STATEOPTIONS._serialized_start=845
-  _STATEOPTIONS._serialized_end=1212
-  _STATEOPTIONS_STATECONCURRENCY._serialized_start=1009
-  _STATEOPTIONS_STATECONCURRENCY._serialized_end=1113
-  _STATEOPTIONS_STATECONSISTENCY._serialized_start=1115
-  _STATEOPTIONS_STATECONSISTENCY._serialized_end=1212
-  _CONFIGURATIONITEM._serialized_start=1215
-  _CONFIGURATIONITEM._serialized_end=1388
-  _CONFIGURATIONITEM_METADATAENTRY._serialized_start=772
-  _CONFIGURATIONITEM_METADATAENTRY._serialized_end=819
+  _globals['_HTTPEXTENSION']._serialized_start=87
```

Comment:
This is indicative of a very different version `grpcio-tools`.

Reply:
I just checked against version 1.60 and that seems to be using `_globals` whereas 1.54 does not. That would be fine for me, but please confirm what version you actually used.

Reply:
When you generated this, did you also get a line like `# Protobuf Python Version: 4.25.0` in the output? Feel free to leave that in.

Reply:
@berndverst Looks like the build passed. Do you need me to look at the above queries?